frontend/src/components/chats/components/current-chat/styles.module.scss

```diff
@@ -14,7 +14,9 @@
 
 .currentChatContent {
   display: flex;
-  flex-grow: 1;
+  flex-direction: column;
+  height: calc(100vh - 235px);
```

Comment:
Can't we set it in one place? In wrapper for example

Reply:
In chat list it is required because when you select a chat the current chat section starts reloading. Before that reloading for a little moment of time it may blink and the section with all chats becomes shorter, it does not look good.
About moving height into wrapper - that is good point, but if our chat has just started and there are not much messages, the height of the chat will be like the height of all messages, which is not good

Reply:
When I wrote about blink, I meant the moment when the Loader appears and then the height of chat list (on the left) decreases, if the height in chatsList is defined - it looks OK